configs/common/packages.yaml

```diff
@@ -139,3 +139,7 @@
       version: [4.7.0]
     udunits:
       version: [2.2.28]
+    openmpi:
```

Comment:
So my hope was that we could entirely avoid building MPI libraries as part of the spack stack and always require the following three things to be present (1) compiler (2) MPI library (3) Python interpreter. But if that's not a reasonable goal, I can be convinced.